src/api/script/api_test_in_spec.sh

```diff
@@ -63,15 +63,12 @@ EOF
 /usr/sbin/memcached $MEMCACHED_USER -l 127.0.0.1 -d -P $MEMCACHED_PID_FILE || exit 1
 
 # migration test
```

Comment:
So please can you enlighten me what the difference is to [rake db:structure:verify](https://github.com/openSUSE/open-build-service/blob/master/src/api/lib/tasks/databases.rake#L87) we run on travis?

There we 

* have the 2.5 dump in [db/migrate/001_initial_database.rb](https://github.com/openSUSE/open-build-service/blob/master/src/api/db/migrate/001_initial_database.rb)
* stripped of the `BTREE` and `schema_migrations` changes
* check if running `db:migrate` produces changes

it is exactly the same thing.

* Why would you run this again? 
* And if you insist of running this again, why don't you at least run the rake task instead of this?